conda-store-server/conda_store_server/_internal/settings.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import functools
+from typing import Any, Callable, Dict
+
+import pydantic
+from sqlalchemy.orm import Session
+
+from conda_store_server import api
+from conda_store_server._internal import schema
+
+
+class Settings:
```

Comment:
This is currently part of the private API - is there a scenario in which these settings would be needed by e.g. nebari?

Reply:
I think this should be private. Users are able to interact with settings using the public [`CondaStore`](conda-store-server/conda_store_server/conda_store.py) object. For example, plugins or other consumers can do stuff like

```
def do_something(self, conda_store) -> str:
    . . .
    conda_store.get_setting("conda_command", namespace=namespace, environment=environment)
   . . . 
```